pkg/monitoring/hyperconverged/rules/alerts/operator_alerts.go

```diff
@@ -79,5 +81,17 @@ func operatorAlerts() []promv1.Rule {
 				healthImpactAlertLabelKey: "critical",
 			},
 		},
+		{
+			Alert: unsupportedArchitecturesAlert,
+			Expr:  intstr.FromString("kubevirt_hco_dict_with_no_supported_architectures == 1"),
+			Annotations: map[string]string{
+				"description": "the {{ $labels.data_import_cron_name }} DataImportCronTemplate (for the {{ $labels.data_source_name }} DataSource) has no supported architectures, and cannot be used to deploy the corresponding DataImportCron.",
```

Comment:
I would say `The {{ $labels.data_import_cron_name }} DataImportCronTemplate (for the {{ $labels.data_source_name }} DataSource) has no supported architectures, this golden image will not be available for use in the cluster`.

Reply:
WDYT about this option?

`"The cluster has no nodes with architecture matches to the architectures of the {{ $labels.data_import_cron_name }} DataImportCronTemplate (for the {{ $labels.managed_data_source_name }} DataSource)."`